src/System Application/App/Upgrade Tags/src/UpgradeTags.Page.al

```diff
@@ -0,0 +1,34 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Upgrade;
+page 9985 "Upgrade Tags"
```

Comment:
Not sure why it is complaining about the already existing ID. The page doesn't exist and the ID isn't used in 25.x. We'll do a swift investigation!

Reply:
No firsts without small hiccups 😝

Reply:
So, it turns out that ID is taken in 25: https://github.com/microsoft/BCApps/blob/releases/25.x/src/System%20Application/App/Word%20Templates/src/WordTemplatesRelatedList.Page.al

Now, the page is deprecated, it isn't used and it's not extensible, so I'd be fine with deleting it. You would have to update your PR with that change.

Reply:
Yeps no problem. Deleted that page 😃